nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+
+function numbers(req, res) {
+    const firstNumber = Number(req.query.a);
+    const secondNumber = Number(req.query.b);
```

Comment:
Is this correct? Did you test it?
How will you get the first and second number from a and b?